### PR TITLE
Updating dougs section of the white paper

### PR DESCRIPTION
**Pull request by amsos**
I've kept dougs piece below mine to compare the two.  would love to get Doug's thoughts on as well

**Description**
Edited and worked on incoporating @zerodoug section and his comments.  I wrote out a first draft of it.

**Motivation and Context**
I believe doug's point of how this could become a technocracy is warranted.

**How Has This Been Written?**
I'm highly confident in the ideas of the section, but want to ensure it doesn't read too emotional and that it accurately captures Dougs concern

**Types of changes**
_What types of changes does your contribution introduce? Put an `x` in all the boxes that apply:_
- [x] Editing a current section
- [ ] Proposing a new section
- [ ] Suggested change to the entire themes of the paper

## Checklist:

- [x] My contritbution follows style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **README** document.

### CODE REVIEW

**Comment by amsos** [comment]
@zerodoug would love to know your thoughts on my additions.

File: f-technological-building-blocks.md [hunk 1]
```diff
@@ -70,6 +70,22 @@ The idea of making governance “writable” is what we’re working towards at
 
 ## Open Source Governance vs Technocracy
 
+There is an unfortunate story that tends to play out when new tools are built that should allow new entrants into the space.  It goes as follows:
```
Comment (christinevandev) [inline f-technological-building-blocks.md:73]: @amsos I like this intro.

File: f-technological-building-blocks.md [hunk 2]
```diff
@@ -70,6 +70,22 @@ The idea of making governance “writable” is what we’re working towards at
...
+1. New tools built to bring new entrants into the space
+2. The learning curve on the tools is high
+3. Those that are already in the space take the time to learn them
+4. New entrants never have a chance to utilize the new tools as they never made it over the learning curve
+
+Even when we build tools that should create more transparency, we sometimes only embolden the status quo.  This can be seen in the way knowledge is disseminated as well.  What it comes to technology tools, we see techno-native individuals grasp new concepts more quickly than those the tools were intended to help.
+
+This becomes a big problem when it has to do with governance and democracy. Rather than a democracy, it enables a Technocracy.
+
+Open Source Governance aims to fix this by making all the content created open for anyone to fork, use, and modify.  The issue is if only "experts" know how to run that process, that we haven't actually changed a power dynamic.  Further, just because you can fork and modify a governance model or policy, doesn't mean you fork the user set or network.
```
Comment (christinevandev) [inline f-technological-building-blocks.md:83]: Or, created a new power dynamic that is different but still not better.  E.g. current power dynamic is politicians and lobbyists. Next one might be tech-native constituents. Sometimes creating a solution to one problem creates a new problem.

I didn't follow the part about "open source governance aims to fix this..." Was 'this' referring to lack of transparency?

Is there a non-jargon way to explain the implications of not being able to fork a user set?

File: f-technological-building-blocks.md [hunk 3]
```diff
@@ -70,6 +70,22 @@ The idea of making governance “writable” is what we’re working towards at
...
+
+Open Source Governance aims to fix this by making all the content created open for anyone to fork, use, and modify.  The issue is if only "experts" know how to run that process, that we haven't actually changed a power dynamic.  Further, just because you can fork and modify a governance model or policy, doesn't mean you fork the user set or network. 
+
+You can build the most elegant new governance model, but due to real world barriers, you could still have trouble getting others to use your system.
+
+Now there are ways to help curtail this, such as a use of elections to ensure we create a rotation of who has the time and access to those tools.  Or ensuring all component governance pieces need to be "composable", so that the changing cost when a network moves from one governance model to a "forked and improved version" is negligible.  But that doesn't change that inherent risk of creating a technocracy when introducing new technology tools.  What we can say is that we're aware of this very real risk and are ensuring we take steps with the distrobution model for the tools we build.
```
Comment (christinevandev) [inline f-technological-building-blocks.md:87]: Liked this incorporation of Doug's points about elections. It fits well imo.

Comment (christinevandev) [inline f-technological-building-blocks.md:87]: I think we need to be more robust with the ending/due diligence or say more is coming soon. Some thoughts like:
What would technology gone right look like:
- seeking out partnerships with academia to have the tech and cases studies be studied early on
- having an advisory panel that has no vested interest in said tech company
- establishing an internal team on said tech company focused on this issue
- determining early indicators of red flags and green flags related to open source governance with metrics on system health

**Review by christinevandev — COMMENTED** [review]
Left a few comments @amsos @zerodoug

File: f-technological-building-blocks.md (hunk 2 above)
Comment (amsos) [reply f-technological-building-blocks.md:83]: That's a good callout that maybe it'd be a new power dynamic that just different.  Will add that

Comment (amsos) [reply f-technological-building-blocks.md:83]: The part on "open source governance aims to fix this..." refers to the idea that anyone can own previous work that is done via forking.  Such that if one group consolidates too much power, if another group has a more fair power dynamic setup, they can take the code (i.e. policy) and fork it into their own...

hmm... need to think of a less jargonny way

File: f-technological-building-blocks.md (hunk 3 above)
Comment (amsos) [reply f-technological-building-blocks.md:87]: ooo these are all intersting.... not sure how to add these elegantly though.. hmmm

File: f-technological-building-blocks.md (hunk 2 above)
Comment (amsos) [reply f-technological-building-blocks.md:83]: The part on "open source governance aims to fix this..." refers to the idea that anyone can own previous work that is done via forking. Such that if one group consolidates too much power, if another group has a more fair power dynamic setup, they can take the code (i.e. policy) and fork it into their own...

hmm... need to think of a less jargonny way

**Comment by amsos** [comment]
@christinevandev to see the updated changes that address some of her comments.

**Comment by zerodoug** [comment]
I like the revised text. I agree that a more specific set of things being explored is better than the intention statement. I think something that open source governance could do in its own development is include laymen in the creation/development process. We naturally build system that suit our persona. Open source communities have been particularly prone to build for themselves. So much so it is a part of the OS ethos. But that tech centric design model is what drives the technocratic bias IMO. 

The only thing I would add is, involving a broader group of people in the design process or rigorously building diverse personas for the system and designing with them in mind could offset the natural technocratic tendency.

**Comment by amsos** [comment]
I like the idea of adding something about expanding those that design with us.

I'm thinking we make a quick PR with that edit.

**Pull request merged**